libbeat/processors/add_process_metadata/gosiger_cid_provider.go

```diff
@@ -0,0 +1,74 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
just realized that this file says `gosiger`. Can you rename it to `gosigar_cid_provider.go`?

Reply:
There is a flipped char, gosig***a***r :)

Reply:
Nice catch, my bad :)